pouch.go

```diff
@@ -250,9 +257,14 @@ func (p *pouch) Run(ctx context.Context) error {
 		select {
 		case <-nextUpdate:
 			log.Printf("Updating secret '%s'", s.Name)
-			err = p.resolveSecret(s.Name, p.Secrets[s.Name])
+			retry, err := p.resolveSecret(s.Name, p.Secrets[s.Name])
 			if err != nil {
-				return err
+				if retry {
+					log.Println(err)
+					<-time.After(SecretRetryPeriod)
```

Comment:
When this period of time expires, it will try to resolve the FilesUsing, does it will use the current secret (not updated)? 
OTOH as `ttu` has not being changed, will the next iteration of for loop occur after `time.After(SecretRetryPeriod)`  ?

Reply:
Yes, it will use the same secret as it will continue being the one with sooner TTU. And yes, it will happen just after `SecretRetryPeriod`.

Reply:
Umm, ok, I see what you meant, this can trigger a loop of restarts, I'm going to do these retries here in place.